testsuite/integration/src/test/java/org/keycloak/testsuite/adapter/AdapterTest.java

```diff
@@ -126,6 +126,11 @@ public void testLoginSSOMax() throws Exception {
         testStrategy.testLoginSSOMax();
     }
 
+    @Test
```

Comment:
Adapter tests are not the correct place to test this. Rather use LoginTest or something else from the base tests inside the new Arquillian testsuite. Please remove all changes to the adapter tests inside testsuite/integration